app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -79,7 +82,7 @@ def entity_ui_name_or_all(entity)
     if entity
       provider = MAPPABLE_ENTITIES[entity]
       model = case entity
-              when 'Vm'
+              when 'Vm', 'VmAzure'
```

Comment:
The problem is that this break the pluggable providers approach. But it seems to be the case before, so if @martinpovolny is ok with this, I guess we can merge. 

We should refactor this though next, so this generic code doesn't know anything about k8s, AWS or Azure

Reply:
See proposed refactor followup: https://github.com/ManageIQ/manageiq-ui-classic/pull/3791/commits/22a532fe4bfcf847ab01d3227d4139c2bfac4fa5
(might also help reviewing this PR — the explicit values there were computed by actually running logic from `entity_ui_name_or_all` and `label_tag_mapping_add` in this PR).

It still won't be "pluggable", there is currently no mechanism by which providers report which types can be mapped in their refresh (and UI shouldn't offer types that wouldn't work).

I'm happy to later do a bigger followup with a migration to a better schema, but I'm afraid the answer at this point will be "don't bother", justifiably so...